cdc/scheduler/internal/tp/agent.go

```diff
@@ -185,7 +175,6 @@ func (a *agent) handleMessage(msg []*schedulepb.Message) []*schedulepb.Message {
 			response := a.handleMessageHeartbeat(message.Heartbeat.GetTableIDs())
 			result = append(result, response)
 		case schedulepb.MsgUnknown:
```

Comment:
What about the following ?

```suggestion
		default:
```